exporters/geneva/include/opentelemetry/exporters/geneva/metrics/data_transport.h

```diff
@@ -11,12 +11,18 @@ OPENTELEMETRY_BEGIN_NAMESPACE
 namespace exporter {
 namespace geneva {
 namespace metrics {
-using ByteVector = std::vector<unsigned char>;
+
+enum class MetricsEventType : uint16_t {
+  ULongMetric = 50,
```

Comment:
Add a link to the original type definition?

Reply:
The original link is MS internal, have added reference to the ksy file where the enum is originally defined.